src/python/gudhi/differentiation/tensorflow.py

```diff
@@ -0,0 +1,251 @@
+import numpy               as np
+import tensorflow          as tf
+import tensorflow_addons   as tfa
```

Comment:
tensorflow_addons needs do be added to the cmake and CI files.

Reply:
I am adding it to CI in #425, but after that you will still need to add the cmake stuff.

Reply:
great thanks!!

Reply:
Actually, now it looks like tensorflow_addons is not used anymore. Should we remove it from gudhi-deploy?